groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
I can't help but feel as though we're going to come back to this conversation one day, but for the moment I'm going to close this issue. It's not because I think that this is better or worse than what we have, it's because I don't want this to be a sticking point, and a decision needs to be made one way or another to allow us to move on.

Relevant: https://youtu.be/NPlMcUxFOlY?t=30m19s